projects/ngrx.io/content/guide/store/recipes/custom-elements.md

```diff
@@ -0,0 +1,65 @@
+# Using Store with Angular Elements
+
+## Creating Angular Elements
+
+Using Store with Angular Elements is similar to using it with standard Angular Component.
```

Comment:
I would remove this statement and move up the first few sentences in the tutorial section to here. I would also use the word recipe instead of tutorial.